packages/next/src/server/response-cache/index.ts

```diff
@@ -8,16 +8,16 @@ import type {
 } from './types'
 import { RouteKind } from '../future/route-kind'
 
-import RenderResult from '../render-result'
 import { Batcher } from '../../lib/batcher'
 import { scheduleOnNextTick } from '../../lib/scheduler'
+import { fromResponseCacheEntry, toResponseCacheEntry } from './utils'
 
 export * from './types'
 
 export default class ResponseCache implements ResponseCacheBase {
   private readonly batcher = Batcher.create<
     { key: string; isOnDemandRevalidate: boolean },
-    ResponseCacheEntry | null,
+    IncrementalCacheItem | null,
```

Comment:
The `IncrementalCacheItem` is sharable as it uses a `string` for the `html` part and not a `RenderResult` which under the hood uses a `ReadableStream` which is consumed on read.